networking/eth2/src/test/java/tech/pegasys/teku/networking/eth2/ActiveEth2P2PNetworkTest.java

```diff
@@ -255,6 +255,56 @@ void onSyncStateChanged_shouldNotResultInMultipleSubscriptions() {
     verify(eventChannels, times(1)).subscribe(eq(BlockGossipChannel.class), any());
   }
 
+  @Test
+  void isCloseToInSync_shouldCalculateWhenDistanceOutOfRange() {
+    final UInt64 currentEpoch = storageSystem.combinedChainDataClient().getCurrentEpoch();
+    final int distanceOutsideRange =
```

Comment:
i'd probably be setting up a `maxFollowDistanceSlots` in the overall test framework. That makes all of this a lot clearer, then you could just be setting the current slot to `maxFollowDistanceSlots.increment()`